app/style/main.scss

```
@@ -116,15 +106,15 @@ $duration: .2s;
     }
   }
 
-  .cookie-icon.default {
+  .cookie__icon.default {
     background-image: url(../assets/cookie.svg);
   }
 
-  .cookie-icon.blue {
-    animation: blue-cookie 5s;
+  .cookie__icon.blue {
```

This would be an example of a modified cookie icon.
```suggestion
  &__cookie-icon--blue {
```